### PR TITLE
Add extension configs for [ tsconfig, jest, webpack, webpack-folder ]

### PR DESCRIPTION
**Pull request by sneera-p**
_**Fixes #IssueNumber**_

**Changes proposed:**

- [x] Add
- [x] Delete
- [ ] Fix
- [ ] Prepare

### CODE REVIEW

**Comment by sneera-p** [comment]
the 'tsconfig.build.json' file exists in nestjs projects. Will be nice to have the tsconfig icon in it instead of the standard json icon

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -2238,13 +2238,8 @@ export const extensions: IFileCollection = {
     {
       icon: 'jest',
       extensions: [
-        'jest.config.json',
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedExtensions.ts:2241]: Is `jest.config.cjs` a real option for this file? If so, could you provide some documentation about it?

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

File: src/iconsManifest/supportedFolders.ts [hunk 2]
```diff
@@ -688,7 +688,13 @@ export const extensions: IFolderCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
-    { icon: 'webpack', extensions: ['webpack'], format: FileFormat.svg },
+    { 
+      icon: 'webpack', 
+      extensions: [
+        'webpack',
+        '.webpack'
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedFolders.ts:695]: What's this `.webpack` extension? is it a filename? If it is, then this is wrong.

Comment (robertohuertasm) [reply src/iconsManifest/supportedFolders.ts:695]: I'm going to remove it to ship the changes to tsconfig. Let's do another PR for this, if needed.

**Review by robertohuertasm — DISMISSED** [review]

**Pull request merged**